funfact/vectorization.py

```diff
@@ -51,17 +51,21 @@ def view(fac, tsrex_scalar, instance: int):
             The original, un-vectorized tensor expression.
         instance (int):
             Index along the vectorization dimension.
+        append (bool):
```

Comment:
While I understand it is better to reuse the `append` keyword to ensure API consistency here, there is a similar semantic ambiguity problem as above in the loss function.

At least, we can change this docstring line to say something like 'indicates whether the vectorization dimension was appended or prepended'.

Reply:
To have a consistent API, we can also call the argument 'vectorize_along_last' everywhere? Slight difference in meaning when it is an active action ('vectorize`) or details about input (`loss`) but maybe that's fine?

Reply:
I have updated the doc string, but kept the name.